libafl/src/events/centralized.rs

```diff
@@ -666,6 +668,9 @@ where
                     log::debug!("[{}] {} was discarded...)", process::id(), event_name);
                 }
             }
+            Event::Stop => {
+                *state.should_stop_mut() = true;
```

Comment:
I think we should have a method that requests to stop, and another one that looks if stop was requested, not a mutable value here

Reply:
ok let's do ``state.request_stop`` and ``state.should_stop``

Reply:
Hmm `request_stop` implies it can fail so let's just do `initiate_stop`?

Reply:
Well it's a request, in theory the fuzzer can ignore it

`request_stop` and `stop_requested`?

Reply:
request_stop, stop-requested & discard_stop_request?